qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local-chrome.properties

```diff
@@ -1,9 +1,9 @@
-webapp.url=http://localhost:8080/openmrs
+webapp.url=https://qa-refapp.openmrs.org/openmrs
```

Comment:
Hey @kdaud , basing on @Muta-Jonathan information above, this report button takes almost 1 hour to run on localhost, which is the other way around switching on qa-server, i think because the github actions doesnt receive much traffic as of now, We can go with qa which makes it faster. whats your opinion.?